Sp02 unconsciousness fix

YetheSamartaka opened the pull request:
Attempt to fix Sp02 behavior by adding another condition to StableVitals.

YetheSamartaka committed:
Sp02 unconsciousness fix

Attempt to fix Sp02 behavior by adding another condition to StableVitals. In this current state, it drops error with missing ) on line 20 of fnc_hasStableVitals

YetheSamartaka committed:
Spo2 unconsciousness fix + tweaks

This should fix Spo2 unconsciousness. Also containts tweaks for gaining/loosing Spo2 values and pneumothorax default value

Tomcat-SG commented on addons/breathing/stringtable.xml:
```diff
@@ -264,6 +264,24 @@
+        </Key>
+		<Key ID="STR_kat_breathing_DESCRIPTION_STABLE_SPO2">
+            <English>After reaching more than selected value (default 85), only then patient can get conscious</English>
+            <German>Wenn mehr als der ausgewählte Wert (Standard 85) erreicht ist, kann der Patient erst dann bewusst werden</German>
```
```suggestion
            <German>Wenn mehr als der ausgewählte Wert (Standard 85) erreicht ist, kann der Patient erst dann Bewusstsein erlangen</German>
```

Tomcat-SG commented on addons/breathing/script_component.hpp:
```diff
@@ -15,3 +15,4 @@
 #endif
 
 #include "\x\kat\addons\main\script_macros.hpp"
+#include "\z\ace\addons\medical_engine\script_macros_medical.hpp"
```
Remove this line after expanding the macros
```suggestion
```

Tomcat-SG commented on addons/breathing/CfgFunctions.hpp:
```diff
@@ -8,8 +8,8 @@ class CfgFunctions {
     class overwrite_medicalStatus {
-        tag = "ace_medical_status";
-        class ace_medical_status {
+        tag = "ace_medical_Status";
```
Why this is made uppercase?

Tomcat-SG reviewed (changes requested):
Also update the changes of hasStableVitals, which were discussed in Discord and why in CfgFunctions `class ace_medical_Status {` is an uppercase? Pls also fix this!

YetheSamartaka committed:
progress

It has issues with ace zeus module - toggle unconsciousness (it gives no errors), it won't wake them up.

YetheSamartaka committed:
remove include script_macros_medical.hpp

Tomcat-SG commented on addons/breathing/functions/fnc_hasStableVitals.sqf:
```diff
@@ -1,7 +1,6 @@
-#include "script_component.hpp"
```
Pls leave this in the file

Tomcat-SG reviewed (changes requested):
pls fix this. after that we are ready to go :)

YetheSamartaka committed:
missing include fix
